.github/workflows/ci.yml

```diff
@@ -45,8 +45,7 @@ jobs:
       env:
         CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_TOKEN }}
       run: |
-        pip install codacy-coverage
-        python-codacy-coverage -r coverage.xml
+        bash <(curl -Ls https://coverage.codacy.com/get.sh) report -r coverage.xml
```

Comment:
This seemed to work locally if I make a test token for my own test Codacy project